r/rules_rust/build_info.json

```diff
@@ -12,5 +12,8 @@
 
     "0.26.*" : {
        "build_script" : "rules_rust_ubi_8.7.sh"
+    },
+    "0.28.0" : {
```

Comment:
@sumitd2 Let's use `0.28.*` so for any future minor release version currency build can use this new script.
Also please update version (row5) and build_script (row8), so any future version can by default will try running with new script you are adding if not matching pattern found.